src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/ConnectionString.cs

```diff
@@ -20,6 +20,8 @@ public class ConnectionString
         private Dictionary<string, string> _keyValuePairs;
         private string _connString;
         private StringBuilder _parseErrorSb;
+        private string DEFAULT_TENANTID = "microsoft.onmicrosoft.com";
```

Comment:
I was told that the tenantId was PII and so I made the change.
@hovsepm can you confirm?